app/helpers/report_helper.rb

```diff
@@ -32,4 +32,46 @@ def sortable(column, title = nil)
 
     link_to title, {sort: column, dir: direction}, class: css_class
   end
+
+  def report_state_description(state)
+    case state
+    when "Submitted"
+      content_tag(:p, content_tag(:strong, "Submitted: ") + "This report has been submitted and is now ready to be validated.") + content_tag(:p, content_tag(:i, "", class: 'fa fa-warning') + "The report may continue to be edited, but changing certain fields may require the report to be submitted again")
+    when "In progress"
+      content_tag(:p, content_tag(:strong, "In progress: ") + "This report is in progress and has not yet been submitted")
+    when "Validated"
+      content_tag(:p, content_tag(:strong, "Validated: ") + "This report has been successfully validated." + content_tag(:p, content_tag(:i, "", class: 'fa fa-warning') + "The report may continue to be edited, but changing certain fields may cause the report to become unvalidated"))
+    when "Returned"
+      content_tag(:p, content_tag(:strong, "Returned: ") + "This report has been returned by a validator and requires updating before it can be submitted again")
+    end
+  end
+
+  def format_search_value_for_display(key, value)
+    case key
+    when "agencies"
+      value = value.map {|v| Agency.find(v).name }
```

Comment:
Perhaps this can be changed to `Agency.where(id: v).pluck(:name).join(",")`? Same for users below